quickstart-query.ts

```diff
@@ -20,15 +22,26 @@ const client: WeaviateClient = await weaviate.connectToWeaviateCloud(
  }
 )
 
-// Check client status
-// console.log(await client.isReady())
+// Run a fetch objects query
+async function fetchObjects() {
+  const questionCollection = client.collections.get('Question')
+
+  const response = await questionCollection.query.fetchObjects({
+    limit: 5,
+    returnMetadata: ['creationTime']
```

Comment:
The quickstart doesn't use the distance measure and the distances aren't in the sample results on the page. Let's leave them out here.